examples/docs/content/docs/extensions/mdx-deck.mdx

```diff
@@ -0,0 +1,56 @@
+import Notification from '../../../src/components/notification.js';
+import {Heading} from '../../../src/components';
+
+# MDX Deck
+
+<Notification type="warning">
```

Comment:
I like this!

![image](https://user-images.githubusercontent.com/2344137/44932681-4baab100-ad1b-11e8-89c8-9cbd7cc0fdaf.png)

We started the convo about icons in #66 and couldn't think of a use-case for them yet, but having an icon like this in the `Notification` title would be a good use case for supporting icons.

![image](https://user-images.githubusercontent.com/2344137/44932733-7268e780-ad1b-11e8-974b-5d0bb9a411f1.png)

> screenshot from [Feather icons](https://feathericons.com/)